zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/LearnerHandler.java

```diff
@@ -649,6 +683,14 @@ public void run() {
         }
     }
 
+    /**
+     * Tests need not send marker packets as they are only needed to
+     * log quorum packet delays
+     */
+    protected boolean shouldSendMarkerPacketForLogging() {
```

Comment:
Did you miss to port changes to tests?
This method is not overridden in this changeset.

Reply:
You have sharp eyes! Thanks!

Reply:
Can you remind me what's the purpose of having this function, why we need to overwrite it in test? Will it break the existing test without set it to false?